### ISSUE TITLE
Configuration Property "ButtlerSpyglass.loglevel" is not defined

### ISSUE THREAD

**Issue by Freighar** [ptarmiganlabs/butler-spyglass#35]
**Describe the bug**
Hello, i followed the instruction using node.js but when i run "node.js" i get an error. I renamed the config file "Production.yaml" but i cannot make it work.

**To Reproduce**
I just installed using npm and then run node index.js.

**Expected behavior**

**Screenshots**
![Error](https://user-images.githubusercontent.com/20541541/107936862-750b3600-6f83-11eb-97c0-bafc4b24a406.PNG)

**Describe environment:**
 - OS: Windows server2 012
 - Containerisation: No
 - Version of Butler Spyglass used: Last
 - Command used to start Butler Spyglass: node index.js

**Config file(s)**
What's the content of the config file(s) you use?
Please make sure to remove all sensitive information before posting it here.

**Additional context**

**Comment by mountaindude** [comment]
Ah, maybe you didn't set the NODE_ENV environment variable..?

I totally agree the instructions could be better there. 
The [Docker section](https://github.com/ptarmiganlabs/butler-spyglass#run-using-docker) does mention it, but it's not mentioned in the [Run from command line](https://github.com/ptarmiganlabs/butler-spyglass#run-from-command-line) section.

Butler Spyglass' sibling project [Butler](https://github.com/ptarmiganlabs/butler) has a way more comprehensive doc site at [butler.ptarmiganlabs.com](https://butler.ptarmiganlabs.com).

There you find a [thorough explanation](https://butler.ptarmiganlabs.com/docs/getting-started/setup/which-config-file/#select-which-config-file-to-use) of how the config file naming works.

To sum it up:
If your config file is called `production.yaml`, you should set the NODE_ENV environment variable to `production`. 
See the [Butler docs](https://butler.ptarmiganlabs.com/docs/getting-started/setup/which-config-file/#setting-environment-variables) for examples.

**Comment by Freighar** [comment]
Yeah that made it work!

thanks a lot!
